backend/plugin/db/mysql/role.go

```diff
@@ -2,39 +2,388 @@ package mysql
 
 import (
 	"context"
+	"database/sql"
 	"fmt"
+	"regexp"
+	"strconv"
 	"strings"
 
-	"github.com/pkg/errors"
-
+	"github.com/bytebase/bytebase/backend/common"
 	"github.com/bytebase/bytebase/backend/plugin/db"
 	"github.com/bytebase/bytebase/backend/plugin/db/util"
+	"github.com/bytebase/bytebase/backend/plugin/parser"
 	storepb "github.com/bytebase/bytebase/proto/generated-go/store"
 )
 
+type roleFind struct {
```

Comment:
It might be difficult to couple both plugin (MySQL driver specific) and storage specific stuff.